src/utils/useWeb3Connect.js

```diff
@@ -326,23 +343,29 @@ function useWeb3Connect() {
   };
   const updateBalance = async (
```

Comment:
More a personal preference I guess, but this function is used before it is defined.